### PR TITLE
Add a Creative Corporea Spark, which provides its items by copying them

### PR DESCRIPTION
**Pull request by Alwinfy**

### CODE REVIEW

File: Common/src/main/java/vazkii/botania/common/item/ModItems.java [hunk 1]
```diff
@@ -304,6 +304,7 @@
 	public static final Item sparkUpgradeIsolated = new ItemSparkUpgrade(/*ModPatterns.SPARK_ISOLATED, */defaultBuilder(), SparkUpgradeType.ISOLATED);
 	public static final Item corporeaSpark = new ItemCorporeaSpark(defaultBuilder());
 	public static final Item corporeaSparkMaster = new ItemCorporeaSpark(defaultBuilder());
+	public static final Item corporeaSparkCreative = new ItemCorporeaSpark(defaultBuilder());
```
Comment (Hubry) [inline Common/src/main/java/vazkii/botania/common/item/ModItems.java]:
```suggestion
	public static final Item corporeaSparkCreative = new ItemCorporeaSpark(defaultBuilder().rarity(Rarity.EPIC));
```
For consistency with all the creative mode items.

**Comment by williewillus** [comment]
Did using an enum { NORMAL, MASTER, CREATIVE } not work out? Having the possibility to get into both master + creative is a bit weird

**Comment by Alwinfy** [comment]
Mostly I just didn't want to break backwards compatability

**Comment by williewillus** [comment]
sure

**Pull request merged**